bgpd/bgp_packet.c

```diff
@@ -2728,14 +2768,6 @@ static int bgp_capability_msg_parse(struct peer *peer, uint8_t *pnt,
 				"%s CAPABILITY has action: %d, code: %u, length %u",
 				peer->host, action, hdr->code, hdr->length);
 
-		if (hdr->length < sizeof(struct capability_mp_data)) {
```

Comment:
why is this no longer necessary?

Reply:
it's moved under MP cap handling, not removed.